src/Symfony/Component/String/Resources/functions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String;
+
+function g(string $string = ''): GraphemeString
```

Comment:
Any reason why we are not passing the `locale` value to these helpers?
Also, would you consider having a full name for them?